vertexai/snippets/text_input_test.go

```diff
@@ -0,0 +1,36 @@
+// Copyright 2024 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package snippets
+
+import (
+	"bytes"
+	"fmt"
+	"strings"
+	"testing"
+
+	"github.com/GoogleCloudPlatform/golang-samples/internal/testutil"
+)
+
+func TestTextInput(t *testing.T) {
+	tc := testutil.SystemTest(t)
+	buf := &bytes.Buffer{}
+	err := textInput(buf, tc.ProjectID, "us-central1", "gemini-1.0-pro-vision-001")
+	if err != nil {
+		t.Errorf("unexpected error: %v", err)
+	}
+	if got := buf.String(); !strings.Contains(got, "positive") {
```

Comment:
Can we deterministically expect this string to be found in the output?

Reply:
Refactored